### ISSUE TITLE
模型微调

### ISSUE THREAD

**Issue by zeroTohero4539** [vl2g/PatentLMM#2]
请问该模型的微调所需要的硬件资源大概是什么样的？

**Comment by 0xnakul** [comment]
We leverage LoRA modules to enhance training efficiency. The original model was trained using 3 A100 GPUs (40GB), but 2 A100s should also work well: the gradient accumulation steps can be adjusted to get the same effective batch size as mentioned in the paper.
